### PR TITLE
Remove MS Legacy Edge support

### PR DESCRIPTION
**Pull request by clintfred**
And update version of recrypt-wasm-binding

There was also some cruft left over from when IE 11 support was removed a while back. That is also removed here.

Tested on the following via browserstack

| Platform | Browser | Notes | Worked? |
| ----------- | ----------- | ------- | ------------ |
| Mac/Big Sur | Chrome 85 | | YES |
| Mac/High Sierra | Safari 11.1 | | YES |
| Win 10 | Edge 18 | Error code 206 on submit password | NO |
| Win 10 | Edge 91 |  | YES |
| Win 10 | IE 11 | Blank page. Console: "Promise is undefined" | NO |
| Win 10 | Safari 5.1 | Blank page | NO |
| Win 10 | Opera 76 | | YES |
| Win 10 | Yandex.12 | UI comes up. Initialize fails with "No WASM support" | NO |
| Win 7 | IE 11 | Blank page. Console: "Promise is undefined" | NO |
| iOS/iPhone 12 | Safari | | YES |
| Android/S8 | Android Browser | | YES |

### CODE REVIEW

File: src/frame/worker/crypto/aes/tests/index.test.ts [hunk 1]
```diff
@@ -276,7 +264,6 @@ describe("AES", () => {
         });
 
         it("uses polyfill API if native is not available", () => {
-            spyOn(CryptoUtils, "isNativeCryptoSupported").and.returnValue(false);
```
Comment (coltfred) [inline src/frame/worker/crypto/aes/tests/index.test.ts:279]: Seems like this test should be defunct right?

File: src/frame/worker/crypto/aes/tests/index.test.ts [hunk 2]
```diff
@@ -212,7 +203,6 @@ describe("AES", () => {
...
```
Comment (coltfred) [inline src/frame/worker/crypto/aes/tests/index.test.ts:215]: Seems like this test should be defunct right?

File: src/frame/worker/crypto/aes/tests/index.test.ts [hunk 3]
```diff
@@ -137,7 +132,6 @@ describe("AES", () => {
...
```
Comment (coltfred) [inline src/frame/worker/crypto/aes/tests/index.test.ts:140]: Seems like this test should be defunct right?

**Review by coltfred — CHANGES_REQUESTED** [review]
It seems like a few tests that were using the `isNativeCryptoSupported` as false should be removed as well as the commented out code that @giarc3 pointed out.

**Comment by clintfred** [comment]
@cjyar Is there an easy way to remove the build caches? I made a new version of the (temporary) recrypt-wasm-binding-0.6.0 zip file, and it not happy. If not I can upload a new one with a different name.

**Comment by clintfred** [comment]
> @cjyar Is there an easy way to remove the build caches? I made a new version of the (temporary) recrypt-wasm-binding-0.6.0 zip file, and it not happy. If not I can upload a new one with a different name.

I uploaded a new artifact, so that should allow it to build now.

**Comment by cjyar** [comment]
> @cjyar Is there an easy way to remove the build caches? I made a new version of the (temporary) recrypt-wasm-binding-0.6.0 zip file, and it not happy. If not I can upload a new one with a different name.

I don't know what build cache it would be using. I don't see any caching actions in the workflows. Are you talking about this error?

> error https://drive.google.com/uc?export=download&id=1gSv3s59BCo38wASfp3_-P2vhPiz0psLY: Integrity check failed for "@ironcorelabs/recrypt-wasm-binding" (computed integrity doesn't match our records, got "sha512-D0Nb4MJdhEJRFbu3j1cU8p5b4lissWiQ2qvzHM8SzuvDpu3V95FmZhjoKS0faamPqY9d58JzCY4A62DvBx1tCQ== sha1-9ciguH3Z6WevJP5gxpSwW0LwYDA=")

That looks like a yarn error that could be caused by a few different things. I don't think our GitHub runners have a local cache, so it's more likely a problem with `yarn.lock`. [This post](https://www.seancdavis.com/blog/fix-yarn-integrity-check-failed/) has some advice.

**Review by skeet70 — APPROVED** [review]
No notes, good ripping.

**Pull request merged**